tests/handlers/records-query.spec.ts

```diff
@@ -988,7 +1472,7 @@ export function testRecordsQueryHandler(): void {
         expect(replyToAliceQuery.status.code).to.equal(200);
         expect(replyToAliceQuery.entries?.length).to.equal(1);
 
-        // actual test: bob should not be able to see unpublished record
+        // actual test: bob should not be able to see unpublished record without published filter
```

Comment:
Sanity: why add the "without published filter"? It seems to imply that bob should be able to see unpublished record if `published` is explicitly set to `false`?

Reply:
Ah yeah I should remove that. I was going to add a negative test here but It's already covered somewhere else with the intent more clear for that case.

I'll revert to:
```suggestion
        // actual test: bob should not be able to see unpublished record
```